github_api/voting.py

```diff
@@ -31,8 +31,10 @@ def get_votes(api, urn, pr):
         if vote and vote_owner != pr_owner:
             votes[vote_owner] = vote
 
-    # by virtue of creating the PR, the owner casts his vote as 1
-    votes[pr_owner] = 1
+    # by virtue of creating the PR, the owner casts their vote as 1
+    # unless they later vote against it, in which case we respect that
+    if votes.get('pr_owner') != -1:
```

Comment:
Thanks, I was just about to point that out. 

This line in its current form actually just allows the GitHub user named "pr_owner" to override the actual PR owner's vote. Also, I feel it's worth pointing out that as of the moment of this review, there are 23 other people who voted for this PR without noticing this mistake.